semiphemeral/twitter.py

```diff
@@ -316,3 +324,34 @@ def delete(self):
                     self.common.session.commit()
 
             self.common.session.commit()
+
+    def import_dump(self, filepath):
+        if self.common.settings.get('delete_tweets'):
+            with open(os.path.join(filepath, 'tweet.js'), 'r') as f:
```

Comment:
With my archive I received this encoding error: `'charmap' codec can't decode byte 0x81`

I fixed it for me by adding the parameter `encoding='UTF-8'` to the `open()` command